### ISSUE TITLE
Add dependency only to dev requirements

### ISSUE THREAD

**Issue by mmsbrggr** [alexander-schranz/sulu-backend-bundle#50]
In the README it's suggested to add the package to the general requirements of a project. However most often, one just want's to add it as a dev-requirement.

**Comment by alexander-schranz** [comment]
yes first I planned to generate as less code as possible so it had requirements to this bundle but I will skip this behaviour and the `--extended` will be the default in the stable release.
